spec/controllers/messages_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe MessagesController, :type => :controller do
+  describe "Get #show" do
+    let!(:user) { create(:user) }
```

Comment:
In these tests they will work fine, but sometimes there are some gotchas to using `let!`. Here's a nice write up on `let` itself and how variables are set in rspec tests: https://www.launchacademy.com/codecabulary/learn-test-driven-development/rspec/before-vs-let